### PR TITLE
Update to 0.13.2

### PR DESCRIPTION
**Pull request by adairrr**
This change publishes 0.13.2 which has updated BOOT definitions and everything.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -63,23 +63,23 @@ abstract-app = { path = "packages/abstract-app" }
 
 # Keep these as path, creates cirular dependency otherwise
 # Only need to re-publish all contracts if a re-publish of abstract-boot is required
-abstract-boot = {path = "packages/abstract-boot" }
+abstract-boot = { path = "packages/abstract-boot", version = "0.13.2" }
```
Comment (CyberHoward) [inline Cargo.toml:66]: why are we setting a version here?

File: packages/abstract-api/Cargo.toml [hunk 2]
```diff
@@ -27,7 +27,7 @@ cw2 = { workspace = true }
 abstract-sdk = { workspace = true }
 abstract-core =  { workspace = true }
 abstract-testing = { workspace = true, optional = true }
-abstract-boot = { version = "0.13.0", workspace = true, optional = true }
```
Comment (CyberHoward) [inline packages/abstract-api/Cargo.toml:30]: and removing it here?

**Review by CyberHoward — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (adairrr) [reply Cargo.toml:66]: Needed a version for publishing... will remove.

File: Cargo.toml [hunk 3]
```diff
@@ -50,8 +50,8 @@ protobuf = { version = "2", features = ["with-bytes"] }
 
 clap = { version = "4.0.32", features = ["derive"] }
 semver = "1.0"
-cw-semver = { version = "1.0"}
-boot-core = { version = "0.10.0" }
```
Comment (CyberHoward) [inline Cargo.toml:54]: Don't want to use the git dep for this, just stick to the release.

**Review by CyberHoward — CHANGES_REQUESTED** [review]

**Pull request closed without merging**